Makefile

```diff
@@ -0,0 +1,28 @@
+CPPFLAGS:=-I./lua/src
+CFLAGS:=--std=c99 -g -Wall -O2 -fPIC
+
+.PHONY: default clean linux-readline lua-linux-readline macos lua-macos
+
+default:
+	@echo "Please do 'make PLATFORM' where PLATFORM is one of these:"
+	@echo "  linux-readline linux-noreadline macos"
+
+clean:
+	cd lua/src && $(MAKE) clean
+	rm -f ./runtime/*.o ./runtime/*.a
+
+
+linux-readline:   runtime/tcore.o lua-linux-readline
```

Comment:
I think we have a runtime phony target, so it will be easier to add another files later.

```
linux-readline:   runtimelib lua-linux-readline 

runtimelib: runtime/tcore.o
```

Reply:
I would prefer adding this when we add those extra files (YAGNI). Unless we can find a way to package all the ".o" into a single file to link against (which is what I was trying to do with the .a)